examples/000_Local_Deployment.ipynb

```diff
@@ -0,0 +1,352 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Local Deployment\n",
+    "\n",
+    "This example steps through how to deploy a local version of DTBase for development purposes. These instructions are a simplication of the developer docs and more details can be found there. \n",
```

Comment:
Maybe add here a mention of what is the new thing that these examples bring, that is not in the dev docs.